src/index.html

```diff
@@ -11,6 +11,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="search__form search__form_big"
+      action="search"
+      method="get"
+      data-qa="big"
+    >
+      <div class="form__div form__div_big">
+        <img 
+          class="form__icon"
+          src="images/Search.svg" 
+          alt="Search icon"
+        >
+        <input 
+          class="form__input form__input_big"
```

Comment:
```suggestion
    <form 
      class="search__form"
      action="search"
      method="get"
      data-qa="big"
    >
      <div class="form__div">
        <img 
          class="form__icon"
          src="images/Search.svg" 
          alt="Search icon"
        >
        <input 
          class="form__input"
```
lets make it work like this, + remove repeated styles